types/treblle/index.d.ts

```diff
@@ -0,0 +1,28 @@
+//=========================================================================================================
+//
+//  ######  #####    #####  #####   ##      ##      #####        ######  ##    ##  #####   #####   ####
+//    ##    ##  ##   ##     ##  ##  ##      ##      ##             ##     ##  ##   ##  ##  ##     ##
+//    ##    #####    #####  #####   ##      ##      #####          ##      ####    #####   #####   ###
+//    ##    ##  ##   ##     ##  ##  ##      ##      ##             ##       ##     ##      ##        ##
+//    ##    ##   ##  #####  #####   ######  ######  #####          ##       ##     ##      #####  ####
+//
+//=========================================================================================================
```

Comment:
There is no need for Word Wall

You must include package name , version,
contributor name , github user, and other stuff.
Please use dts-gen to generate the files for you to eliminate manual labour and look to [base64-js] (https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/base64-js) package for reference